packages/compass-saved-aggregations-queries/src/stores/delete-item.ts

```diff
@@ -0,0 +1,77 @@
+import type { Reducer } from 'redux';
```

Comment:
This follows the [feature folders](https://redux.js.org/style-guide/style-guide#structure-files-as-feature-folders-with-single-file-logic) recommendation: all actions related to delete are in one place